### ISSUE TITLE
CopyQ opens a blank uncloseable window on first start

### ISSUE THREAD

**Issue by bortoliniamm** [hluk/CopyQ#1400]
I am using CopyQ v 3.11.1 on Windows 10.

Every time I open the application from my Desktop, its window doesn't open, I have to go to the notification area and click it. 

![image](https://user-images.githubusercontent.com/65280344/83333951-8c927980-a2a3-11ea-8f89-5001e299b9e5.png)

When I do that, I get a blank uncloseable window on the start (see below). 

![image](https://user-images.githubusercontent.com/65280344/83333834-e2b2ed00-a2a2-11ea-9a77-577e9310c163.png)

Then, I have to go to the notification area again and right-click CopyQ icon, then I have to click the **Show/Hide** option (see below).

![image](https://user-images.githubusercontent.com/65280344/83333910-4fc68280-a2a3-11ea-840d-ce83a5d7fd70.png)

Only after that, the main window is shown correctly (see below).

![image](https://user-images.githubusercontent.com/65280344/83333983-cb283400-a2a3-11ea-8d79-4377a2da5ea3.png)

Is there any configuration that would fix that?

Thanks

**Comment by hluk** [comment]
This looks like duplicate of #1228. Possibly an issue with item preview dock and newer Qt framework.

You can try using build with Qt 5.9: https://ci.appveyor.com/project/hluk/copyq/builds/32745642/artifacts
